client/state/selectors/get-reader-recommended-sites.js

```diff
@@ -1,10 +1,12 @@
-/**
- * Returns the recommended sites for a given seed
+/***
+ * Returns the recommended sites for a given seed.
  *
  * @param  {Object} state  Global state tree
  * @param {Number} seed the elasticsearch seed for which to grab recs
- * @return {Array} Reader Sites
+ * @return {Object} { recommendedSites: [ Reader Sites ], offset:  Current paging offset }
  */
-export default function getReaderRecommendedSites( state, seed ) {
-	return state.reader.recommendedSites[ seed ];
-}
+const getReaderRecommendedSites = ( state, seed ) => ( {
```

Comment:
or maybe expose it as two selectors so we're not creating new objects in a selector?